supplier_api_tester/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.2.3
```

Comment:
**question:** why not 2.2.2?

Reply:
@giorgosT  2.2.2 was just released from a different branch.